docs/audits.md

```diff
@@ -1348,6 +1348,31 @@ tell which changes of the subsequent release the pinned commit includes.
 Change the `#!yaml uses:` clause to pin the action using a SHA reference
 which points to a Git tag.
 
+## `unpinned-container-images`
+
+| Type     | Examples                | Introduced in | Works offline  | Enabled by default | Configurable |
+|----------|-------------------------|---------------|----------------|--------------------|--------------|
+| Workflow, Action  | [unpinned-container-images.yml] | v1.7.0        | ✅            | ✅                | ❌          |
+
+Checks for `container.image` values where the image version is not specified:
+
+```yaml
+container:
+  image: foo/bar
+```
+and where that version is `latest`:
+
+```yaml
+container:
+  image: foo/bar:latest
+```
+
+### Remediation
+
+Change the `#!yaml container.image:` value to pin the container image to a specific, static tag
+or even better, to a SHA256 reference of the image.
```

Comment:
I fully intend to flesh this out with more detailed content once we reach consensus on the desirable behavior